frontend/src/components/SamplesList.tsx

```diff
@@ -62,6 +62,10 @@ interface ISampleListProps {
   userEmail?: string | null;
   setUserEmail?: Dispatch<SetStateAction<string | null>>;
   customToolbarUI?: JSX.Element;
+  exportDropdownItems?: Array<{
+    label: string;
+    columnDefs: ColDef[];
+  }>;
```

Comment:
Each hashmap in this array will appear as a dropdown item under the export button. I chose an array here to give us flexibility when we want to add more export options in the future. Also got Ben's ok with this UI choice.

Example usage:
```ts
exportDropdownItems={[
  {
    label: "Generate Phenotype files for dbGaP",
    columnDefs: DbGapPhenotypeColumns,
  },
]}
```
![CleanShot 2025-04-23 at 15 46 17](https://github.com/user-attachments/assets/e8162783-d34d-47d7-81db-7e629f7cc4be)